src/wazuh_db/schema_global.sql

```diff
@@ -33,6 +33,8 @@ CREATE TABLE IF NOT EXISTS agent (
     date_add INTEGER NOT NULL,
     last_keepalive INTEGER,
     `group` TEXT DEFAULT 'default',
+    group_source TEXT NOT NULL CHECK (group_source IN ('manual', 'remote')) DEFAULT 'manual',
```

Comment:
We should add a third value here "undefined" and use it as the default value. This will let us keep the group source as undefined until it is updated by a a manual or remote source.

Reply:
As we discussed, I renamed to "unknown"